settings-all.lp

```diff
@@ -13,15 +13,16 @@ mg.include('scripts/pi-hole/lua/header_authenticated.lp','r')
 PageTitle = "All Settings"
 mg.include('scripts/pi-hole/lua/settings_header.lp','r')
 ?>
-<div class="row settings-level-2" id="advanced-content">
+<div class="row settings-level-expert" id="advanced-content">
```

Comment:
I don't think we need this class here.

```suggestion
<div class="row" id="advanced-content">
```

This is probably the reason the page is empty on yubi's screenshot.